client/common/cmdline.c

```diff
@@ -3001,18 +3001,18 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 
 			settings->MultifragMaxRequestSize = (UINT32)val;
 		}
-		CommandLineSwitchCase(arg, "max-loop-time")
+		CommandLineSwitchCase(arg, "no-io-timeout")
```

Comment:
Please do not change command line or settings variable names

Reply:
why ?, old name does not reflect current behavior,  and  i am more and more sure what this logic does not required at all, as it is specific to loop which i am eliminated completely, probably this  logic does not needed at all, and timeouts should be handled by user (app which does polling)

Reply:
this is completely eliminated now.